csrc/scheduler/reduction.cpp

```diff
@@ -63,7 +63,7 @@ void reduceProductTo(int64_t& z, int64_t& y, int64_t& x, const int64_t max) {
   }
 }
 
-std::unique_ptr<ReductionParams> innerReductionHeuristic(
+std::unique_ptr<ReductionParams> inner2dReductionHeuristic(
```

Comment:
nitpick: this is renamed as a 2d reduction. But in the code comment, we refer to this as 1D schedule, which is a bit confusing.